File: Models/Authentication/ApiToken.cs
namespace aliment_backend.Models.Authentication
{
    /// <summary>
    /// Représente un jeton d'API comprenant un jeton d'accès et un jeton de rafraîchissement.
    /// </summary>
    public class ApiToken
    {
        /// <summary>
        /// Obtient ou définit le jeton d'accès.
        /// </summary>
        public string? AccessToken { get; set; }

        /// <summary>
        /// Obtient ou définit le jeton de rafraîchissement.
        /// </summary>
        public string? RefreshToken { get; set; }
    }
}

File: src/app/features/auth/_models/apiToken.ts
/**
 * La classe `apiToken` représente un objet utilisé dans l'application pour structurer les données relatives aux jetons d'accès et de rafraîchissement.
 * Ces jetons sont couramment utilisés dans les systèmes d'authentification pour assurer la sécurité et la gestion des sessions.
 */
export class apiToken {
  /**
   * Propriété représentant le jeton d'accès (Access Token) associé à l'objet `ApiToken`.
   * @type {string}
   */
  accessToken!: string;

  /**
   * Propriété représentant le jeton de rafraîchissement (Refresh Token) associé à l'objet `ApiToken`.
   * @type {string}
   */
  refreshToken!: string;
}
